client/my-sites/customer-home/cards/features/help-search/index.jsx

```diff
@@ -66,14 +65,18 @@ const HelpSearch = ( props ) => {
 				<div className="help-search__inner">
 					<CardHeading>{ translate( 'Get help' ) }</CardHeading>
 					<div className="help-search__content">
-						<div className="help-search__search">
+						<div className="help-search__search inline-help__search">
```

Comment:
I don't think it's good that we're using classnames from another component here. This couples this component _tightly_ to another component. This makes is vulnerable to a whole class of visual bugs. Is there a way around this?

I t

Reply:
there isn't a simple solution for this but it's totally doable. Maybe the best option could be to create a new <HelpSearch /> component and consume it from <InlineHelp /> and <HelpSearch /> components.
Another option trying to tackle it changing the CSS rules, but again it will require many changes making this PR even bigger.

I know it isn't perfect but It worths imo. What do you think about creating a new issue for this and start to work on once it lands?

Reply:
I say we ship as is and I'll create an Issue to track the need to extract a shared component. Then we can refactor out these tight couplings.